### PR TITLE
Repositories have been added for each entity

### PR DESCRIPTION
**Pull request by RainbowInHell**

### CODE REVIEW

File: Car_Rental.DLL/Repositories/VehicleRepository.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+using Microsoft.EntityFrameworkCore;
+
+namespace CarRental.DLL.Repositories
+{
+    public class VehicleRepository : GenericRepository<Vehicle>, IVehiclesRepository
+    {
+        public VehicleRepository(CarRentalContext context) : base(context)
+        { }
+
+        public IEnumerable<Vehicle> GetUnRentedVehicles()
```
Comment (KatsiarynaAnashkevich) [inline Car_Rental.DLL/Repositories/VehicleRepository.cs:12]: Think about async/await here(and not only here)

File: Car_Rental.DLL/EntitiesConfigurations/ManufacturerEntityConfiguration.cs [hunk 2]
```diff
@@ -9,17 +9,17 @@ public class ManufacturerEntityConfiguration : IEntityTypeConfiguration<Manufact
         public void Configure(EntityTypeBuilder<Manufacturer> manufacturerEntityBuilder)
         {
             manufacturerEntityBuilder
-                .HasKey(e => e.Id);
+                .HasKey(x => x.Id);
 
             manufacturerEntityBuilder
-                .Property(e => e.Name)
+                .Property(x => x.Name)
                 .HasColumnType("varchar")
```
[the remark this answers is not in the document]
Comment (RainbowInHell) [reply Car_Rental.DLL/EntitiesConfigurations/ManufacturerEntityConfiguration.cs:16]: At the moment I'm still thinking about it. I'll make a decision later

File: Car_Rental.DLL/Repositories/VehicleRepository.cs (hunk 1 above)
Comment (KatsiarynaAnashkevich) [reply Car_Rental.DLL/Repositories/VehicleRepository.cs:12]: Got it!

File: Car_Rental.DLL/EntitiesConfigurations/ManufacturerEntityConfiguration.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (RainbowInHell) [reply Car_Rental.DLL/EntitiesConfigurations/ManufacturerEntityConfiguration.cs]: *

File: Car_Rental.DLL/Repositories/VehicleRepository.cs (hunk 1 above)
Comment (RainbowInHell) [reply Car_Rental.DLL/Repositories/VehicleRepository.cs]: *

**Review by KatsiarynaAnashkevich — APPROVED** [review]

**Pull request merged**